spec/index.bs

```diff
@@ -3533,9 +3533,13 @@ The {{GPUMapMode}} flags determine how a {{GPUBuffer}} is mapped when calling
 <span id=texture-interface></span>
 </h3>
 
-Issue: Remove this definition: <dfn dfn>texture</dfn>
+A <dfn dfn>texture</dfn> is a {{GPUTextureDimension/1d}}, {{GPUTextureDimension/2d}}, or
```

Comment:
Technically textures are a collection of such arrays (one per subresource). And it might be useful to note that the order of element in the array may be optimized for multidimensional access and not linear (like you said in the comment)

Reply:
Thanks, done! (The next paragraph starts to define the subresources, so I only tweaked this paragraph so say "1d, 2d, or 3d array*s*")